main/ZgatewayBT.ino

```diff
@@ -300,7 +300,7 @@ void MiScaleDiscovery(char* mac) {
       //component type,name,availability topic,device class,value template,payload on, payload off, unit of measurement
   };
 
-  createDiscoveryFromList(mac, MiScalesensor, MiScaleparametersCount);
+  createDiscoveryFromList(mac, MiScalesensor, MiScaleparametersCount, "", "Xiaomi", "");
```

Comment:
You can put MISCALE for the model in this case

Reply:
fixed